src/operation/emulation_model.jl

```diff
@@ -356,19 +355,22 @@ function calculate_dual_variables!(model::EmulationModel)
 end
 
 """
-The one step solution method for the emulation model. Custom EmulationModel need to reimplement
-this method. This method is called by run! and execute!.
+The one step solution method for the emulation model. Any Custom EmulationModel
+needs to reimplement this method. This method is called by run! and execute!.
 """
 function one_step_solve!(model::EmulationModel)
     jump_model = get_jump_model(model)
-    JuMP.optimize!(jump_model)
+    log = get_solve_timed_log(model)
+    _, log[:timed_solve_time], log[:solve_bytes_alloc], log[:sec_in_gc] =
+        @timed JuMP.optimize!(jump_model)
     model_status = JuMP.primal_status(jump_model)
     if model_status != MOI.FEASIBLE_POINT::MOI.ResultStatusCode
         error("Optimizer returned $model_status")
-    else
-        calculate_aux_variables!(model)
-        calculate_dual_variables!(model)
     end
+
+    _, log[:timed_calculate_aux_variables] = @timed calculate_aux_variables!(model)
+    _, log[:timed_calculate_dual_variables] = @timed calculate_dual_variables!(model)
```

Comment:
I added these, but I don't think anything is logging them.

Reply:
For these tests, there aren't any. We need to extend the test cases to account for these.